themes/mariana-reference.json

```diff
@@ -246,10 +273,35 @@
 			}
 		},
 		{
-			"name": "User-defined constant",
+			"name": "Function type",
+			"scope": "support.type",
+			"settings": {
+				"fontStyle": "italic"
+			}
+		},
+		{
+			"name": "User-defined constant character placeholder Python",
+			"scope": [
+				"meta.fstring.python constant.character.format.placeholder"
+			],
+			"settings": {
+				"foreground": "LIGHT_1"
+			}
+		},
+		{
+			"name": "Class",
+			"scope": [
+				"entity.name.type.class"
```

Comment:
this duplicates `entity.name` which is already orange

Reply:
Not really, this overrides `entity.name.type` which is blue & italic, but in Python classes it should be orange & upright